### PR TITLE
Feature: Session lifetime config in resource policy

### PR DESCRIPTION
**Pull request by lizable**
## Description
This PR resolves a new configuration `max_session_lifetime`, which can be applied in resource policy.
Related: [manager#583](https://github.com/lablup/backend.ai-manager/pull/583)
> ⚠️ NOTE: 
> - Value "0" in `max_session_lifetime` means Infinite(♾) likewise in `idle_timeout`.
> - This feature is available from `22.03`.

## Before
- Create Resource Policy Dialog
Create-Resource-Policy-before

- Update Resource Policy Dialog
Update-Resource-Policy-before

## After
- Create Resource Policy Dialog
Create-Resource-Policy

- Update Resource Policy Dialog
Update-Resource-Policy

### CODE REVIEW

File: src/components/backend-ai-session-launcher.ts [hunk 1]
```diff
@@ -3370,7 +3370,9 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
                 ${this.resource_templates_filtered.map((item) => html`
                   <mwc-list-item value="${item.name}"
                             id="${item.name}-button"
-                            @click="${(e) => {this._chooseResourceTemplate(e);}}"
+                            @click="${(e) => {
+    this._chooseResourceTemplate(e);
+  }}"
```
Comment (adrysn) [inline src/components/backend-ai-session-launcher.ts]: This worsens the readability.

**Review by adrysn — CHANGES_REQUESTED** [review]
Minor code style issues.

File: src/components/backend-ai-session-launcher.ts (hunk 1 above)
Comment (lizable) [reply src/components/backend-ai-session-launcher.ts]: hmm, I guess it's because of "eslint" code styling. I'll revert to the previous code style.

Comment (adrysn) [reply src/components/backend-ai-session-launcher.ts]: Perhaps, we need to update the eslint config not to rearrange the code in a weird way.

File: src/components/backend-ai-session-launcher.ts [hunk 2]
```diff
@@ -3441,7 +3441,9 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
                   <mwc-list-item  class="slider-list-item">
                     <lablup-slider id="mem-resource" class="mem"
                                    pin snaps expand step=0.05 editable markers
-                                   @change="${(e) => {this._applyResourceValueChanges(e); this._updateShmemLimit()}}"
+                                   @change="${(e) => {
+    this._applyResourceValueChanges(e); this._updateShmemLimit();
+  }}"
```
Comment (adrysn) [inline src/components/backend-ai-session-launcher.ts]: Here as well.

File: src/components/backend-ai-session-launcher.ts [hunk 3]
```diff
@@ -3456,7 +3458,9 @@ export default class BackendAiSessionLauncher extends BackendAIPage {
                 <mwc-list-item class="slider-list-item">
                   <lablup-slider id="shmem-resource" class="mem"
                                  pin snaps step="0.0125" editable markers
-                                 @change="${(e) => {this._applyResourceValueChanges(e); this._updateShmemLimit()}}"
+                                 @change="${(e) => {
+    this._applyResourceValueChanges(e); this._updateShmemLimit();
+  }}"
```
Comment (adrysn) [inline src/components/backend-ai-session-launcher.ts]: Here as well.

**Review by adrysn — CHANGES_REQUESTED** [review]
Further minor styling issues. We may need to change the eslint config.

**Review by adrysn — APPROVED** [review]
LGTM.

**Pull request merged**